client/daemon.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+const daemonBinary = "dockerd"
+
+// DaemonProxy acts as a cli.Handler to proxy calls to the daemon binary
+type DaemonProxy struct{}
+
+// NewDaemonProxy returns a new handler
+func NewDaemonProxy() DaemonProxy {
+	return DaemonProxy{}
+}
+
+// CmdDaemon execs dockerd with the same flags
+// TODO: add a deprecation warning?
+func (p DaemonProxy) CmdDaemon(args ...string) error {
+	args = stripDaemonArg(os.Args[1:])
+
+	binaryAbsPath, err := exec.LookPath(daemonBinary)
+	if err != nil {
+		return err
+	}
+
+	return syscall.Exec(
```

Comment:
Ha! Ya know of all the things I did, I never ran docker daemon, just dockerd. I assumed a test covered that. However, Windows does really need the legacy aspect as we haven't hit RTM yet. So I'm tempted to say don't do the exec bit on Windows, error out with a 'run dockerd instead' error message. This is justified even more by the fact we are very shortly about to make dockerd a real Windows service after this PR is merged.